### PR TITLE
expose d on rex lr scheduler for lr_scheduler_args

### PR DESCRIPTION
**Pull request by uwidev**
Lets people train with REX on a less aggressive curve.

As d approaches 0, the scheduler more or less becomes linear, but as it approaches 1 (default set to 0.9), it take much longer until the model starts the finer end of the fine-tune.

See this interactive graph of the REX function and mess around with d.
https://www.desmos.com/calculator/u8erssf5hh

This also enables training on lower steps with REX, as typically with 0.9, there's a very small amount of steps where the LR is small.

### CODE REVIEW

**Comment by Jelosus2** [comment]
This has been already added to the flux branch of the backend a while ago. But is missing on the main branch

**Comment by uwidev** [comment]
Is flux eventually going to be merged into main? If so we can close this.

**Comment by Jelosus2** [comment]
Yes, is gonna be merged so I'm gonna close the PR. Thanks for the interest

**Comment by derrian-distro** [comment]
flux branch has been merged into main, both on the backend, and the frontend
